curricula/templatetags/tracking_pixel_url.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+import urllib
+import random
+
+register = template.Library()
+
+@register.filter(name='tracking_pixel_url')
+
+def tracking_pixel_url(path):
+    # Append a random number so that the tracking_pixel
+    # isn't cached - we want each visit to a url to go
+    # fetch a new tracking pixel image.
+    random_num = random.randint(1,1001)
+    tracking_pixel_url = 'https://studio.code.org/curriculum_tracking_pixel?from=' + urllib.quote_plus(path) + '&r=' + str(random_num)
```

Comment:
Is the random number required for cache-busting if we set the right cache controls on the server?